proxy/envoy/policy.go

```diff
@@ -61,6 +61,7 @@ func insertMixerFilter(listeners []*Listener, instances []*model.ServiceInstance
 							"source.ip":  context.IPAddress,
 							"source.uid": context.UID,
 						},
+						QuotaName: "RequestCount",
```

Comment:
Yeah, I'm not sure that fixing the quota name statically in the envoy is actually what we want long term. But, it's what the mixerclient understands today. We should file an issue to figure out the correct way to handle this and update mixerclient  separately.